src/detectors/ZDC/ZDC.cc

```diff
@@ -82,7 +82,7 @@ extern "C" {
              "EcalFarForwardZDCTruthClusterAssociations"}, // edm4eic::MCRecoClusterParticleAssociation
             {
               .energyWeight = "log",
-              .sampFrac = 1.0,
+              .sampFrac = 0.701,
```

Comment:
For my own understanding: is this a correction to the EM scale? Or is this a scale factor to adjust the relative weighting between the ECal and HCal components?

If it's the latter, this should be a parameter for the Neutron Reconstruction algorithm rather than overloading the sampling fraction parameter for the cluster reconstruction.

Reply:
I determined the value of the sampling fraction by running gamma simulations in the Ecal, and taking the ratio between the generated photon energy and the sum of simulated hit energies.  I think this was set to 1.0 as a placeholder earlier.

Reply:
Gotcha! Makes sense! That being said, I do think including a parameter to adjust the relative weighting of the HCal vs. ECal would be a useful. Something like so:
```
float ecalWeight = 1.0;
float hcalWeight = 1.0;
```

Where, of course, the parameters could be updated when needed. I suggest this since (and correct if I'm wrong), the HCal part of the ZDC is non-compensating, right? So e/h will differ between the ECal and HCal parts...

(Or if it is, this algorithm may get used by a different combination of subsystems where the HCal part is non-compensating...)